integration-cli/docker_utils.go

```diff
@@ -1564,6 +1564,7 @@ func setupRegistry(c *check.C) *testRegistryV2 {
 		if err = reg.Ping(); err == nil {
 			break
 		}
+		c.Log(err)
```

Comment:
not sure, I thought it would be printed in the `c.Assert` below but I was getting a totally different error so putting this here did the magic and showed me the right error (unauthorized). I guess I can move the log before the assert.